freex/src/components/Home/Home.module.css

```diff
@@ -1,17 +1,18 @@
 .container {
-  display: flex;
-  justify-content: space-between;
-  margin-top: 10rem;
-  margin-bottom: 3rem;
+  display: grid;
+  grid-template-areas: "left right";
+  grid-template-columns: 1fr 1fr;
+  padding: 10rem 2rem 2rem 0;
```

Comment:
Masz bardzo duży padding-top - 10 rem - ja bym dała mniej.  Przez to jest taka duża odległośc pomiędzy nawigacją i treścią strony i trzeba scrollować